src/libs/OptionsListUtils.js

```diff
@@ -1043,6 +1054,149 @@ function getTagListSections(rawTags, recentlyUsedTags, selectedOptions, searchIn
     return tagSections;
 }
 
+/**
+ * Transforms tax rates to a new object - to add codes and new name with concatenated name and value.
+ *
+ * @param {Object} policyTaxRates
+ * @returns {Array<Object>}
+ */
+function transformedTaxRates(policyTaxRates) {
```

Comment:
NAB, We can define the type of policyTaxRates structure.
We're currently migrating to TypeScript, so this would be good thing to add